src/kOS/AddOns/Trajectories/TRWrapper.cs

```diff
@@ -38,35 +39,42 @@ private static void init()
                 wrapped = false;
                 return;
             }
-            trGetImpactPosition = trajectoriesAPIType.GetMethod("getImpactPosition");
+            trGetImpactPosition = trajectoriesAPIType.GetMethod("GetImpactPosition");
             if (trGetImpactPosition == null)
             {
```

Comment:
I'd like to attempt to make this work regardless of which version of Trajectories is installed.
Would it be a bad thing if I changed this to look for *both* spellings and use whichever one it finds?
That will make life easier for me later if I choose to make a back-port for RO compatibility.

Reply:
Sounds like a good idea, only bad thing although not a big issue is anyone using an older Trajectories version will not gain the error checking in the newer API.
With the new method `HasTarget` you could just return a message stating something like, 'This function only available in Trajectories v2.x.x'
On a side note I've also created a branch of Trajectoriesv2.x.x for KSP1.3 (not built a release yet but tests have been positive so a release is imminent).

Thinking about it I am going to add a new method to the Trajectories API which returns its version number.

I can make the changes later today if you wish and add them to this PR.

Reply:
I don't think older users will get the error checking, but at least it would still work as well as it did before before the error checking was present, i.e. using new kOS with old Trajectories wouldn't *break*.  By the way I was looking at this myself overnight and was going to send a PR back to you on top of your PR.

Reply:
I'll wait till you send your PR. 😉 
you could use the new API method to check the version if you wish, I've called it `GetVersion` returns a string i.e. `2.1.0`
Also now going to add `GetVersionMajor` , `GetVersionMinor` and `GetVersionPatch` that return an integer value.
Obviously a check for a version earlier than v2.x.x will be just to find if the Trajectories addon is available and if `GetVersion` exists.
Commit will be later today on Trajectories/master I'll let you know when I commit so you can have a look at the code.

Reply:
I had a few questions about the API but it's probably faster to PR back to you what I have and let you handle it than keep asking questions and doing it myself. Let me make the PR. It's not big.

Reply:
So HasTarget didn't exist in older Trajectories versions?  That answers one of the questions I had (did you just happen to add the suffix to kOS it because it was forgotten before, or because it truly was new).

Reply:
`HasTarget` is new to v2.x.x and does not exist in the older versions.

Reply:
Is your branch "Trajectories-v2.0.0-API-update" based on an older version of our ``develop``?  When I try to PR back to you I see a very big list of diffs, far more than the few I made.

Reply:
The PR to your branch has been made.

Reply:
Thanks, will have a look soon 😉